Jacky's Pi Sys/pi-systems_door-subsystem.py

```diff
@@ -1,39 +1,58 @@
 import importlib
+from enum import IntEnum
 subsys = importlib.import_module('pi-systems_subsystem-base')
 comms = importlib.import_module('pi-systems_communications')
-from enum import IntEnum
+
 
 class DoorSubsystem(subsys.Subsystem):
 
     class Procedure(IntEnum):
-        OpenDoor = ord("o")
-        CloseDoor = ord("c")
-
+        OpenDoor = 111
+        CloseDoor = 99
+        setDoorState = 3
+        getDoorState = 4
 
     def __init__(self, name=None, thread_id=None, address=None):
         super().__init__(name=name, thread_id=thread_id, loop_delay_ms=5000)
 
         self.new_state = None
         self.address = address
 
-
     def loop(self):
-        
         with self.lock:
             if self.new_state is not None:
-                print("Door state updating (%s)" % (self.Procedure(self.new_state).name))
+                print("Door state updating (%s)" %
+                      (self.Procedure(self.new_state).name))
 
                 # Check sensors and things
 
                 comms.intra_write(self.address,
-                    self.generate_intra_protocol_message(
-                        action=comms.IntraModCommAction.ExecuteProcedure,
-                        procedure=self.new_state
-                ))
+                                  comms.generate
+                                  (action=comms.IntraModCommAction.
+                                   ExecuteProcedure, procedure=self.new_state))
 
                 self.new_state = None
 
-    
+    # function to get current door state from door control arduino using I2C
+    # return: doorState and doorAngle as a tuple
+    def getCurrentDoorState(self):
```

Comment:
Make sure your variable and function names are in snake_case to conform with pep8 standards

i.e. getCurrentDoorState() should be get_current_door_state()